arm9/overlays/24/src/ov24_02254840.c

```diff
@@ -0,0 +1,254 @@
+#include "global.h"
+#include "bg_window.h"
+#include "unk_0200CA44.h"
+#include "gf_gfx_loader.h"
+#include "overlay_24.h"
+
+extern void Poketch_InitApp(void *func1, void *func2);
+extern u32 ov20_02254130(void *, u32, void *, void*, u32);
+extern void ov20_02254198(u32);
+extern void ov20_02252C14(u32, u32);
+extern void ov20_022529A0(u32);
+extern void ov20_0225298C(u32);
+extern BOOL ov20_02252C08(u32);
+extern BOOL TouchScreen_GetTapState(u32*, u32*);
+
+static void ov24_02254840(void)
+{
+    Poketch_InitApp(ov24_02254854, ov24_02254990);
+}
+
+#define NitroStaticInit ov24_02254840
+#include "sinit.h"
+
+BOOL ov24_02254854(MemoPadAppHandler** appHandlerOut, int arg1, int arg2, int arg3) {
+    MemoPadAppHandler* appHandler = AllocFromHeap(HEAP_ID_POKETCH_APP, sizeof(MemoPadAppHandler));
+    if (appHandler) {
+        if (ov24_0225489C(appHandler, arg1, arg2, arg3) != 0) {
+            if (SysTask_CreateOnMainQueue((SysTaskFunc)ov24_02254918, appHandler, 1) != 0) {
+                *appHandlerOut = appHandler;
+                return TRUE;
+            }
+        }
+        FreeToHeap(appHandler);
+    }
+    return FALSE;
+}
+
+BOOL ov24_0225489C(MemoPadAppHandler* appHandler, u32 arg1, u32 arg2, u32 arg3) {
+    static const u8 ov24_022550F8[] = {
```

Comment:
genearlly these should be outside the function

Reply:
is this a general rule just for consistency? these aren't used anywhere else so my intuition would tell me that they're defined and used exclusively within the functions

Reply:
yeah a general rule for consistency

Reply:
ov24_022550F8 appears to denote the bounding boxes of the pencil and eraser buttons